frontend/src/components/map/MapTooltip.tsx

```diff
@@ -4,11 +4,11 @@ import {
   CardDescription,
   CardHeader,
   CardTitle,
-} from "./ui/card";
-import { Button } from "./ui/button";
-import { Cross, LayoutDashboard, X } from "lucide-react";
+} from ".././ui/card";
+import { Button } from ".././ui/button";
+import { LayoutDashboard, X } from "lucide-react";
 import { Link } from "@tanstack/react-router";
-import { Tooltip, TooltipContent, TooltipTrigger } from "./ui/tooltip";
+import { Tooltip, TooltipContent, TooltipTrigger } from ".././ui/tooltip";
```

Comment:
I'm not sure what the changes from `../ui` to `.././ui` are all about, but okay...